### ISSUE TITLE
Error when specifying config file with -c

### ISSUE THREAD

**Issue by danielo515** [Leanty/tree-gateway#135]
Hello, If I try to run tree-gateway with the `-c` option I get the following error:

`Error starting gateway: Class constructor Configuration cannot be invoked without 'new'`

This is how I am executing it:

`treeGateway -c ./config/tree-gateway.yaml`

**Comment by thiagobustamante** [comment]
Hi, 

This error is generally related to some translation problem during typescript compilation. Can you confirm some informations:

 - Did you install the gateway through the command `npm i -g tree-gateway`?
 - What is your node version? Tree Gateway requires node 8 or greater to run.

**Comment by danielo515** [comment]
Yes, I installed it using the global flag. My node version is 8.9.0

Thanks

**Comment by thiagobustamante** [comment]
The error occur only when you start it specifying the --config argument? If yes, can you send me your config file to help me to reproduce the problem?

**Comment by danielo515** [comment]
Yes, exactly, except that it is the.`-c` option. The file works if I just allow tree-gateway picking it automatically (the file is name tree-gateway.yaml)

**Comment by thiagobustamante** [comment]
I could not reproduce it. Can you please send me your tree-gateway.yaml file?
